docs/guides/using-org-slugs.mdx

```diff
@@ -0,0 +1,397 @@
+      link: "/docs/quickstarts/nextjs",
+      icon: "nextjs",
+    },
+    {
+      title: "Enable organizations for your instance",
+      link: "/docs/organizations/overview",
+      icon: "globe",
+    },
+    {
+      title: "Examine the complete sample repository on Github",
+      link: "https://github.com/clerk/orgs/tree/main/examples/sync-org-with-url",
+      icon: "git",
+    }
+  ]}
+>
+  - Allow users to choose an organization slug
+  - Include an organization slug in your URLs
+  - Render organization-specific content
+</TutorialHero>
+
+## Introduction
+
+In this guide, you will learn how to include an organization slug in your application's URLs, and
+how to render information about the user's active organization at runtime.
+
+## Background
+
+With [Organizations](/docs/organizations/overview), users can create entities that represent their companies within your
+application. Users can be members of multiple organizations, but can only have one
+[active organization](/docs/organizations/overview#active-organization) at any given time. Clerk automatically sets organizations as
+active when a user picks an organization with a pre-built component (e.g. the
+[Organization Switcher](/docs/components/organization/organization-switcher)), and Clerk provides
+APIs (like [<code>auth()</code>](/docs/references/nextjs/auth) and [<code>useAuth()</code>](/docs/references/react/use-auth)), that let you determine the active organization and render the appropriate
+content.
+
+A relatively common practice is to represent organization-scoped spaces within an app by prefixing
+URLs with an organization slug. For example, imagine a B2B application named "Petstore" that has
+two customers, "Acmecorp" and "Widgetco". The application may choose to have urls like:
+
+`https://petstore.example.com/orgs/`<b>`acmecorp`</b>`/dashboard` ⬅ indicates **Acmecorp**'s dashboard
+`https://petstore.example.com/orgs/`<b>`widgetco`</b>`/dashboard` ⬅ indicates **Widgetco**'s dashboard
+
+It is also possible to use an [organization ID](/docs/references/javascript/organization/organization#properties) to
+indicate the active organization in URLs:
+
+`https://petstore.example.com/orgs/`<b>`org_1a2b3c4d5e6f7g8e`</b>`/dashboard` ⬅ **Acmecorp's ID** indicates Acmecorp's dashboard
+
+This guide will focus on using a human-comprehensible slug, but IDs are just as well-supported in Clerk's component
+configuration.
+
+> [!CAUTION]
+> Think carefully about whether you need to include organization slugs in URLs! Consider:
+>
+> - Will organization-specific links be more useful to your users than
+>   organization-agnostic links? If most users will only ever be in one organization (that represents
+>   their company), they won't need the slug to disambiguate the organization when sharing
+>   links with their coworkers. Public documentation, marketing, and third-party blogs are also
+>   made easier to write if links aren't tied to any specific organization.
```

Comment:
Does this feel clear enough to folks? If not, I can add an example. It would be something like:

> For example, if someone is writing a public post that includes a link to settings, if that link is `/orgs/acmecorp/settings`, it will only be useful to employees of Acmecorp! If the URL is just `/settings`, then it's universally applicable and automatically contextualized into the click-er's organization by virtue of the organization encoded on the Clerk session.